themes/aframe/layout/blog.ejs

```diff
@@ -42,7 +71,6 @@
           <span class="float-right"><a href="<%- page_url(nextPost.path) %>" class="guide-link guide-link-right"><%= nextPost.title %> →</a></span>
```

Comment:
feel free to change these to be unescaped (`<%=` -> `<%-`):

![image](https://cloud.githubusercontent.com/assets/203725/14481792/34cc94c8-00eb-11e6-8a73-f9d9ac156e03.png)